### ISSUE TITLE
Training on the custom dataset

### ISSUE THREAD

**Issue by FrozenSilent** [Lakonik/SSDNeRF#24]
Hi! Thanks for releasing the code of your wonderful work, including both training and inference. I want to train SSDNeRF on my own dataset and have prepared the dataset in the format of the shapenet srn dataset. However, after two days of training, the results are still not good. Here are the visualization results of one tri-plane representation.
![scene_0000](https://github.com/Lakonik/SSDNeRF/assets/31409366/8d74a85a-5e7a-4d08-abfe-9adb8a599076)
It can be seen that there are many noises in the feature plane, which also reflects in the final geometry reconstruction.
![image](https://github.com/Lakonik/SSDNeRF/assets/31409366/595cb9b6-3a86-4f89-b385-efccf5f4ee4b)
And the test rendering result is also not good.
![scene_0000_011](https://github.com/Lakonik/SSDNeRF/assets/31409366/de7cb605-aa57-4a72-9e5b-faaa1315905c)
I use the config file [ssdnerf_cars_uncond](https://github.com/Lakonik/SSDNeRF/blob/main/configs/paper_cfgs/ssdnerf_cars_uncond.py) for training. So what should I pay attention to during the training process? Can you give some suggestions based on the current training results? Thank you very much!

**Comment by Lakonik** [comment]
Hi! I think this could be caused by imbalanced diffusion and rendering loss. It is very important in SSDNeRF to tune the relative weights of diffusion and rendering loss. Also, I would recommend starting from the new models (e.g., [ssdnerf_cars_uncond_16bit](https://github.com/Lakonik/SSDNeRF/blob/main/configs/new_cfgs/ssdnerf_cars_uncond_16bit.py)), which is more stable to train.

**Comment by Lakonik** [comment]
Also, I noticed another minor issue: the object seems too small in the triplane space. It is recommended to normalize the objects' scale to fit them in the triplanes. The scale can be altered through the dataloader's `radius` parameter.

https://github.com/Lakonik/SSDNeRF/blob/5150270b37f691fd4fcb6614fcd55caa0bf3940b/lib/datasets/shapenet_srn.py#L44

**Comment by FrozenSilent** [comment]
Thanks for your advice! I did use Blender to generate the dataset and modified the code that generates the rays accordingly. I will try the config file you specified for training. Also thank you very much for your suggestion on the data processing part!

**Comment by FrozenSilent** [comment]
With your suggestion, I tried training again, and this time the noise in the tri-plane was much less. At least I could see the approximate shape of the model clearly, but the surface was still not smooth. 
![scene_0001](https://github.com/Lakonik/SSDNeRF/assets/31409366/09e952be-fa4e-4625-93ce-560f07a3ecdf)
![image](https://github.com/Lakonik/SSDNeRF/assets/31409366/351713f6-8830-4e3e-818d-ad71b8c20a23)
May I ask which parameters you suggest adjusting to improve training results? In addition, I am also trying to fine-tune the pre-trained checkpoint of the shapenet table on my data, and the current results also gradually appear a lot of noise.
![scene_000000](https://github.com/Lakonik/SSDNeRF/assets/31409366/fdfb7969-ae19-4b66-a72c-420ce8ac39ee)

**Comment by harshD42** [comment]
@FrozenSilent could you please explain in detail how you generated your custom shapenet srn data? I can't find any resources for the same. I have a few computationally generated meshes that I would like to convert into the specified data format and I've written a script for it in blender. The training on the said data works but the testing fails. It would be a great help if you could breakdown your process.
